{"name": "test_demo2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == -1", "trace": "self = <test_demo快速体验.Test_Demo object at 0x0000019A7CFC3A10>\n\n    def test_demo2(self):\n        a = 5\n        b = -1\n>       assert a == b\nE       assert 5 == -1\n\ntest_demo快速体验.py:11: AssertionError"}, "start": 1701408934211, "stop": 1701408934211, "uuid": "51a0d15f-7ba8-4070-b854-5937772b91ea", "historyId": "cf7a1c2bcdfe90fd8722425b5493acb9", "testCaseId": "cf7a1c2bcdfe90fd8722425b5493acb9", "fullName": "test_demo快速体验.Test_Demo#test_demo2", "labels": [{"name": "suite", "value": "test_demo快速体验"}, {"name": "subSuite", "value": "Test_Demo"}, {"name": "host", "value": "QuMoSi"}, {"name": "thread", "value": "17248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_demo快速体验"}]}